extensions/api/observability/src/main/java/org/eclipse/dataspaceconnector/api/observability/ObservabilityController.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2021 Daimler TSS GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Daimler TSS GmbH - Initial API and Implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - add negotiation endpoint
+ *
+ */
+
+
+package org.eclipse.dataspaceconnector.api.observability;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.dataspaceconnector.spi.system.readiness.ReadinessService;
+
+
+@Consumes({ MediaType.APPLICATION_JSON })
+@Produces({ MediaType.APPLICATION_JSON })
+@Path("/")
+public class ObservabilityController {
+
+
+    private final ReadinessService readinessService;
+
+    public ObservabilityController(ReadinessService provider) {
+        readinessService = provider;
+    }
+
+    @GET
+    @Path("health")
+    public Response checkHealth() {
+        return getLiveness();
+    }
+
+    @GET
+    @Path("liveness")
+    public Response getLiveness() {
+        return readinessService.isLive() ?
+                Response.ok().build() :
+                Response.status(404).build();
```

Comment:
consider returning 503, as spring actuator does. 5xx also clearly indicates that the request can be retried.